cmd/esc/cli/env_ls.go

```diff
@@ -56,11 +59,13 @@ func newEnvLsCmd(env *envCommand) *cobra.Command {
 
 	cmd.PersistentFlags().StringVarP(
 		&orgFilter, "organization", "o", "", "Filter returned stacks to those in a specific organization")
+	cmd.PersistentFlags().StringVarP(
+		&projectFilter, "project", "p", "", "Filter returned stacks to those in a specific project")
```

Comment:
I know this is just copy from the line above, but this really should be `environments` instead of `stacks` right? (it's probably copypasta from pulumi/pulumi)